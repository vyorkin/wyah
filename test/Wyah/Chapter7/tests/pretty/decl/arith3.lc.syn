Decl "three" (ELam (Var "x") (ELam (Var "f") (EApp (EVar (Var "f")) (EApp (EVar (Var "f")) (EApp (EVar (Var "f")) (EVar (Var "x")))))))
