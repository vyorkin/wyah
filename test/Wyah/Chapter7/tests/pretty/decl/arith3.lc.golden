let three = \x -> \f -> f (f (f x));